docs/ws2812_driver.md

```diff
@@ -33,10 +33,11 @@ The default setting is 280 µs, which should work for most cases, but this can b
 Some variants of the WS2812 may have their color components in a different physical or logical order. For example, the WS2812B-2020 has physically swapped red and green LEDs, which causes the wrong color to be displayed, because the default order of the bytes sent over the wire is defined as GRB.
 In this case, you can change the byte order by defining `WS2812_BYTE_ORDER` as one of the following values:
 
-| Byte order                        | Known devices                 |
-|-----------------------------------|-------------------------------|
-| `WS2812_BYTE_ORDER_GRB` (default) | Most WS2812's, SK6812, SK6805 |
-| `WS2812_BYTE_ORDER_RGB`           | WS2812B-2020                  |
+| Byte order                        | Known devices                         |
+|-----------------------------------|---------------------------------------|
+| `WS2812_BYTE_ORDER_GRB` (default) | Most WS2812's, SK6812, SK6805         |
+| `WS2812_BYTE_ORDER_RGB`           | WS2812B-2020                          |
+| `WS2812_BYTE_ORDER_BGR`           | some custom keyboard PCBs (MatrixLabs)|
```

Comment:
The actual part number of the LEDs on these boards would be better here, if you can find it.

Reply:
They are custom wired PCBs usually with one or more TM1812 RGB controllers.

And the RGB LEDs used can be pretty much anything. Even single LEDs are probably possible.

Reply:
Ah, okay. The part number would be TM1812 then, from the pinout I found on LCSC it seems like it's essentially four WS2811s smushed together. Or are the LEDs not actually wired correctly to the controllers?

Reply:
The LEDs can probably wired to the controller in no particular order.
I don't think that there is a specific required pinout for it since the LEDs that are connected to the controller can be anything.

Reply:
Well, technically you can use whatever colour LEDs you like, but the pinout still designates R, G and B outputs:
![image](https://user-images.githubusercontent.com/4781841/104754747-b1dde480-57ad-11eb-81ea-6702ef9e1b7d.png)
What I want to know is whether the LEDs are wired accordingly, which would indicate that the byte order is in fact BGR internally, or whether it's simply because the red LEDs are connected to the blue outputs and vice versa.

Reply:
for the PCB that I have here it's impossible to tell.

I don't have any schematics for it and I have no idea what kinda of RGB LEDs were used on it so I can't just follow the traces either.